README.md

```diff
@@ -26,20 +26,16 @@ pip install ocean-lib predict-eth ccxt enforce_typing numpy matplotlib
 #add pwd to bash path
 export PATH=$PATH:.
 
-#set judges private key (needed for "getpreds")
-export REMOTE_TEST_PRIVATE_KEY1=<judges key> 
+#set judges private key and Polygon Scan API Key
+export REMOTE_TEST_PRIVATE_KEY1=<judges key>
+export POLYGONSCAN_API_KEY=<api key>
 ```
 
 # CLI
 
 ```text
-Usage: pej getentries|getpreds|getnmses
-
-  Follow these steps:
-  1. pej getentries - from chain, output 1 entries.csv
-  2. pej getpreds CSV_DIR - from 1 entries.csv, output N predvals_NFT_ADDRX.csv
-  3. pej getnmses CSV_DIR - from N predvals*.csv, output 1 nmses.csv
+Usage: pej getnmses START_DATE DEADLINE
```

Comment:
Thinking more about this:
- CLI just needs `DEADLINE`. It doesn't need `START_DATE`
- then later on, you'll look at all txs from t1 = (`DEADLINE` - one week) to t2 = `DEADLINE`

Also: given that pej is now just a single CLI call, you can change from `pej getnmses` to just `pej`